src/elements/content-picker/stories/ContentPicker.mdx

```diff
@@ -0,0 +1,82 @@
+import { Meta } from '@storybook/blocks';
+
+import ContentPickerStories from './ContentPicker.stories';
+
+<Meta of={ContentPickerStories} />
+
+### Demo ([Documentation](https://developer.box.com/docs/box-content-picker))
+
+```jsx
+
+var ContentPicker = require('./ContentPicker').default;
+
+
+<IntlProvider locale="en">
+
+    <ContentPicker
+
+        features={FEATURES}
+
+        rootFolderId={FOLDER_ID}
+
+        token={TOKEN}
+
+        {...PROPS}
+
+    />
+
+</IntlProvider>
+
+```
+
+
+### Screenshot
+
+<img src="https://user-images.githubusercontent.com/1075325/27887156-0940ee3e-6194-11e7-8e22-961139e82dfe.png" width="600"  alt=""/>
+
+---
+
+
+### Props
+
+
+| Prop                           | Type                      | Default          | Description                                                                                                                                                   |
+|--------------------------------|---------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| token*                         | string                    |                  | *See the [developer docs](https://developer.box.com/guides/embed/ui-elements/picker/#Options).*                                                               |
```

Comment:
Do you like using developer.box.com links? Or what about github links? i have no preference either way, just wanted to see what you thought

Reply:
the context of the MDX is from the original readmes, I would use them for now till we can use typescript to auto generate stuff